src/main/scala/com/fulcrumgenomics/umi/GroupReadsByUmi.scala

```diff
@@ -205,19 +205,19 @@ object GroupReadsByUmi {
 
     /** Assigns IDs to each UMI based on the root to which is it mapped. */
     protected def assignIdsToNodes(roots: Seq[Node]): Map[Umi, MoleculeId] = {
-      val mappings = mutable.Buffer[(Umi,MoleculeId)]()
+      val mappings = Map.newBuilder[Umi,MoleculeId]
```

Comment:
presumably we used `mutable.Buffer` for performance?

Reply:
I think we used it because it was the _obvious_ way to build up a map mutably.